core/bin/snapshots_creator/Cargo.toml

```diff
@@ -18,13 +18,13 @@ zksync_dal = { path = "../../lib/dal" }
 zksync_env_config = { path = "../../lib/env_config" }
 zksync_utils = { path = "../../lib/utils" }
 zksync_types = { path = "../../lib/types" }
-zksync_core = { path = "../../lib/zksync_core" }
 zksync_object_store = { path = "../../lib/object_store" }
 vlog = { path = "../../lib/vlog" }
 
 anyhow = "1.0"
 tokio = { version = "1", features = ["full"] }
 tracing = "0.1"
 futures = "0.3"
-serde = { version = "1.0.189", features = ["derive"] }
```

Comment:
btw, how did you spot that some dependencies were unused? Do you have a tool for that?

Reply:
No, I've just manually searched the crate code for mentions of a particular crate.